db/repair_bucket.go

```diff
@@ -273,7 +274,7 @@ func (r RepairBucket) WriteRepairedDocsToBucket(docId string, originalDoc, updat
 
 	doc, err := unmarshalDocument(docId, contentToSave)
 	if err != nil {
-		return backupOrDryRunDocId, fmt.Errorf("Error unmarshalling updated/original doc.  Err: %v", err)
+		return backupOrDryRunDocId, pkgerrors.Wrapf(err, "Error unmarshalling updated/original doc.")
```

Comment:
Should we be wrapping down in unmarshalDocument?

Reply:
Yup, totally agree.  This will give us stack traces for any errors that originate from `unmarshalDocument()` across the codebase.